devel/development.ini.example

```diff
@@ -59,6 +59,10 @@ openid_template = {username}.id.stg.fedoraproject.org
 # The cache warming is annoying during development because it takes so long. Let's disable it by default.
 warm_cache_on_start = false
 
+test_gating.required = True
+# WaiverDB and Greenwave in vagrant are provided via docker local services
+waiverdb_api_url = http://bodhi_user:pass@localhost:6544/api/v1.0
+greenwave_api_url = http://localhost:6545/api/v1.0
```

Comment:
I tried this one now locally and I'm getting this json response when I'm trying to access this URL:
```
{"message":"The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again."}
```

I just did bstartdeps and tried to access the URL.

Did I missed something?

Reply:
Production GW will give you the same response:
`https://greenwave-web-greenwave.app.os.fedoraproject.org/api/v1.0`

Try:
`http://localhost:6545/api/v1.0/about`
`http://localhost:6545/api/v1.0/policies`
`http://localhost:6545/api/v1.0/metrics`

Output should be the same as in production:
`https://greenwave-web-greenwave.app.os.fedoraproject.org/api/v1.0/metrics`
`https://greenwave-web-greenwave.app.os.fedoraproject.org/api/v1.0/about`
`https://greenwave-web-greenwave.app.os.fedoraproject.org/api/v1.0/policies`

Reply:
In this case, everything looks OK